[docs] #159, #171, #179: Update config sections (#213)
hyperledger/iroha-2-docs: src/guide/configure/client-configuration.md

connections. For client configuration, you can set up two addresses:
`TORII_API_URL` and `TORII_TELEMETRY_URL`.

### `TORII_API_URL`

First, the `TORII_API_URL` is the same as `TORII` `API_URL` in the
[peer configuration](peer-configuration.md#api_url). This is the module
responsible for handling incoming and outgoing connections. You should also
add the prefix `http://` or (_preferably_) `https://` to the address. For
example:

```json
"TORII_API_URL": "http://127.0.0.1:8080"
```

### `TORII_TELEMETRY_URL`

The `TORII_TELEMETRY_URL` is used to specify the prometheus endpoint
address. You can set `TORII_TELEMETRY_URL` like this:

```json
"TORII_TELEMETRY_URL": "http://127.0.0.1:8180"
```

A `GET` request to the `127.0.0.1:8180/status` will give you a JSON-encoded
representation of the top-level metrics, while a `GET` request to
`127.0.0.1:8180/metrics` will give you a (somewhat verbose) list of all
available metrics gathered in Iroha. You might want to change this if
you're having trouble gathering metrics using `prometheus`.

::: info

Learn how to [monitor Iroha performance](./../advanced/metrics.md) using
prometheus.

:::

## Transaction Limits

> You can specify the transaction limits to which each transaction must

Reviewer: ```suggestion
You can specify the transaction limits that each transaction must
```
